Packs/CloudConvert/Integrations/CloudConvert/CloudConvert.yml

```diff
@@ -0,0 +1,380 @@
+      - mobi
+      - mod
+      - mov
+      - mp3
+      - mp4
+      - mpeg
+      - mpg
+      - mts
+      - mxf
+      - odd
+      - odp
+      - ods
+      - odt
+      - oeb
+      - oga
+      - ogg
+      - otf
+      - pdb
+      - pdf
+      - png
+      - ppt
+      - pptx
+      - ps
+      - psd
+      - rar
+      - rm
+      - rmvb
+      - rst
+      - rtf
+      - sf2
+      - svg
+      - swf
+      - tar
+      - tar.bz2
+      - tar.gz
+      - tar.lzo
+      - tex
+      - tiff
+      - ts
+      - ttf
+      - txt
+      - vob
+      - voc
+      - wav
+      - weba
+      - webm
+      - webp
+      - wma
+      - wmf
+      - wmv
+      - woff
+      - woff2
+      - wtv
+      - xls
+      - xlsx
+      - xps
+      - zip
+      required: true
+      secret: false
+    deprecated: false
+    description: Convert a priorly uploaded file to desired format
+    execution: false
+    name: CloudConvert-convert
+    outputs:
+    - contextPath: CloudConvert.Task.id
+      description: Unique task id of scanned file
+      type: String
+    - contextPath: CloudConvert.Task.status
+      description: Status of current task
+      type: String
+    - contextPath: CloudConvert.Task.message
+      description: Response message from API
+      type: String
+    - contextPath: CloudConvert.Task.operation
+      description: The operation conducted
+      type: String
+    - contextPath: CloudConvert.Task.result.files.filename
+      description: Resulted file name
+      type: String
+    - contextPath: CloudConvert.Task.result.files.size
+      description: Resulted size of file
+      type: Number
+    - contextPath: CloudConvert.Task.result.files.url
+      description: Resulted file url
+      type: String
+    - contextPath: CloudConvert.Task.created_at
+      description: Creation Time
+      type: Date
+    - contextPath: CloudConvert.Task.started_at
+      description: Operation start time
+      type: Date
+    - contextPath: CloudConvert.Task.ended_at
+      description: Operation end time
+      type: Date
+    - contextPath: CloudConvert.Task.host_name
+      description: Name of host used for operation
+      type: String
+    - contextPath: CloudConvert.Task.storage
+      description: Storage server for operation
+      type: String
+    - contextPath: CloudConvert.Task.depends_on_task_ids
+      description: The ID of the previous operation conducted on this file
+      type: String
+    - contextPath: CloudConvert.Task.links
+      description: APIs link for operation
+      type: String
+  - arguments:
+    - default: false
+      description: Task id of the convert task of the file of interest
+      isArray: false
+      name: task_id
+      required: true
+      secret: false
+    - default: false
+      description: use this argument if you are checking on an export to war room
+        entry operation, so an entry will be created if the operation is finished
+      isArray: false
+      name: is_entry
```

Comment:
I agree the name could be clearer, but it should be clear that this is regarding an operation that has already been requested - meaning, this indicated 'are we checking on an export to entry operation', rather than 'check on this operation and also return it as file', so I think return_as_file could be a bit confusing. maybe export_to_entry_operation?

Reply:
@Noy-Maimon Let's remember to talk about this argument with our technical writer for the integration